src/Symfony/Component/DependencyInjection/Compiler/RegisterAutoconfigureAttributesPass.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Attribute\Autoconfigure;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+/**
+ * Reads #[Autoconfigure] attributes on definitions that are autoconfigured
+ * and don't have the "container.ignore_attributes" tag.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+final class RegisterAutoconfigureAttributesPass implements CompilerPassInterface
+{
+    private $ignoreAttributesTag;
+    private $registerForAutoconfiguration;
+
+    public function __construct(string $ignoreAttributesTag = 'container.ignore_attributes')
```

Comment:
should probably be ignored in `UnusedTagsPass`

Reply:
we don't use to anywhere, so no need to